third_party/airflow/armada/operators/armada.py

```diff
@@ -131,3 +139,12 @@ def _get_lookout_url(self, job_id: str) -> str:
         if self.lookout_url_template is None:
             return ""
         return self.lookout_url_template.replace("<job_id>", job_id)
+
+    def render_template_fields(
```

Comment:
Can you add this method to `ArmadaDeferrableOperator` as well? It'd be nice to keep feature parity between the two.

Reply:
Will do! I'll add that and the unit tests.